utility/heroku-release.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+if os.environ.get("HEROKU_APP_NAME", "").startswith("sfdoc-review-apps-"):
+    assert not os.system("pip install --upgrade -r requirements/test.txt")
+    os.environ["DJANGO_SETTINGS_MODULE"] = "config.settings.integration_tests"
```

Comment:
you could use the `--ds` option for pytest instead. https://pytest-django.readthedocs.io/en/latest/configuring_django.html#command-line-option-ds-settings

I'll leave this up to you but it seems a bit silly to have heroku-release.sh call into Python just to shell back out to run other commands...